src/styles/box/body.scss

```diff
@@ -0,0 +1,134 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  width: $card-width;
```

Comment:
The variable `$card-width` is used but not defined in this file. Ensure that it is defined elsewhere in your SCSS files, or define it here to avoid compilation errors.